tests/test_schemas.py

```diff
@@ -1,73 +1,174 @@
+import pytest
+
 from starlette.applications import Starlette
 from starlette.endpoints import HTTPEndpoint
 from starlette.schemas import OpenAPIResponse, SchemaGenerator
 from starlette.testclient import TestClient
 
-app = Starlette()
-app.schema_generator = SchemaGenerator(
-    {"openapi": "3.0.0", "info": {"title": "Example API", "version": "1.0"}}
-)
```

Comment:
It'd be nice if we could keep a smaller change-footprint. It's hard to accurately review, with a big diff like this.

Reply:
I know, I was trying to keep it minimal. But adding more routes to the same 2 tests looked more difficult to maintain. Happy to revert the test refactor.